### PR TITLE
docs: Added foreign key action callback syntax example

### PR DESCRIPTION
**Pull request by konnsim**
Ran into this today and found the solution in an old AnswerOverflow forum post.
https://www.answeroverflow.com/m/1173301063642787880

Figure it should be in the docs.

### CODE REVIEW

**Comment by realmikesolo** [comment]
Hey @konnsim. Thank you for your help! Could you please update the code again because now there is `foreignColumns` instead of `references`:

```ts
import { foreignKey, pgTable, serial, text, integer } from 'drizzle-orm/pg-core';

export const users = pgTable('users', {
  id: serial('id').primaryKey(),
  name: text('name'),
});

export const posts = pgTable(
  'posts',
  {
    id: serial('id').primaryKey(),
    name: text('name'),
    author: integer('author').notNull(),
  },
  (table) => ({
    fk: foreignKey({
      name: 'author_fk',
      columns: [table.author],
      foreignColumns: [users.id],
    })
      .onDelete('cascade')
      .onUpdate('cascade'),
  }),
);
```

**Comment by konnsim** [comment]
No worries, let me know if there's anything else :)

File: src/content/documentation/docs/rqb.mdx [hunk 1]
```diff
@@ -403,6 +403,31 @@ export const posts = pgTable('posts', {
...
+	id: serial('id').primaryKey(),
+	name: text('name'),
+});
+
+export const posts = pgTable('posts', {
+	id: serial('id').primaryKey(),
+	name: text('name'),
+	author: integer('author').notNull(),
+}, (table) => {
+	fk: foreignKey({
```
Comment (realmikesolo) [inline src/content/documentation/docs/rqb.mdx:421]: It's missing a return statement in the callback function.

```ts
// ...
 (table) => ({
    fk: foreignKey({
      name: 'author_fk',
      columns: [table.author],
      foreignColumns: [users.id],
    })
      .onDelete('cascade')
      .onUpdate('cascade'),
  }),
```

**Review by realmikesolo — CHANGES_REQUESTED** [review]
Hey @konnsim. Thank you. One more change and we will merge it

File: src/content/documentation/docs/rqb.mdx (hunk 1 above)
Comment (konnsim) [reply src/content/documentation/docs/rqb.mdx]: done, I've fixed the indentation and added a trailing comma to match the other examples as well.

**Pull request merged**